Source/JavaScriptCore/offlineasm/arm64.rb

```diff
@@ -227,6 +250,94 @@ def arm64Operand(kind)
     end
 end
 
+class VecRegisterID
+    def arm64Operand(kind)
+        case @name
+        when 'v0'
+            arm64VecName('v16', kind)
```

Comment:
Why are these named this way? Perhaps we should use _b, _h, _s, etc.... When I hacked on this, I did it in the instructions themselves, but I think this is better.